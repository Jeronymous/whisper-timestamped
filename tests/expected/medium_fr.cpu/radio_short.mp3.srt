1
00:00:30,800 --> 00:00:35,600
Le plus important au poker ce ne sont pas les cartes, c'est ce que vous en faites.

2
00:00:36,160 --> 00:00:38,820
Winamax, la référence du poker en ligne.

3
00:00:44,940 --> 00:00:51,360
Bonsoir à toutes et tous, vous êtes sur BFM TV, nous sommes en direct, c'est bien sûr BFM story avec tout ce qui fait l'actualité.

4
00:00:52,000 --> 00:00:56,100
Durant 60 minutes ce sont des gros plans, des analyses, des réactions que nous vous proposons.

5
00:00:56,100 --> 00:01:03,960
Comment Eric Verth peut-il encore soutenir la réforme des retraites alors qu'il est englué dans sa propre affaire, l'affaire Verth-Bettancourt?

6
00:01:03,960 --> 00:01:07,200
Question posée par les leaders de la CFDT et la CGT.

7
00:01:07,400 --> 00:01:16,380
Réponse de Nicolas Sarkozy, Eric Verth portera le débat sur les retraites, on en parle dans BFM story avec le numéro de la CFDT.

8
00:01:16,780 --> 00:01:23,140
Et puis il y a une bataille qui a démarré, celle entre Marine Le Pen et Bruno Gognich, la bataille de la succession de Jean-Marie Le Pen à la tête du Front National.

9
00:01:23,280 --> 00:01:28,920
La tournée de campagne de Marine Le Pen commence aujourd'hui dans le Var, Marine Le Pen sera en direct dans BFM story.

10
00:01:28,980 --> 00:01:32,260
Restez avec nous Marine Le Pen dans moins de 3 minutes, à tout de suite.

11
00:01:32,260 --> 00:01:33,460
Musique

12
00:01:46,680 --> 00:01:53,540
L'actualité c'est aussi aujourd'hui un dernier adieu, dernier adieu à Laurent Fignon, c'était au cimetière du Père Lachaise à Paris.

13
00:01:54,020 --> 00:02:00,600
L'ancien double vainqueur du Tour de France, vaincu par le cancer à 50 ans, a été incinéré en petit comité aujourd'hui.

