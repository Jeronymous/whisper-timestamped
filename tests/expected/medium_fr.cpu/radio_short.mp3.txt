Le plus important au poker ce ne sont pas les cartes, c'est ce que vous en faites.
Winamax, la référence du poker en ligne.
Bonsoir à toutes et tous, vous êtes sur BFM TV, nous sommes en direct, c'est bien sûr BFM story avec tout ce qui fait l'actualité.
Durant 60 minutes ce sont des gros plans, des analyses, des réactions que nous vous proposons.
Comment Eric Verth peut-il encore soutenir la réforme des retraites alors qu'il est englué dans sa propre affaire, l'affaire Verth-Bettancourt?
Question posée par les leaders de la CFDT et la CGT.
Réponse de Nicolas Sarkozy, Eric Verth portera le débat sur les retraites, on en parle dans BFM story avec le numéro de la CFDT.
Et puis il y a une bataille qui a démarré, celle entre Marine Le Pen et Bruno Gognich, la bataille de la succession de Jean-Marie Le Pen à la tête du Front National.
La tournée de campagne de Marine Le Pen commence aujourd'hui dans le Var, Marine Le Pen sera en direct dans BFM story.
Restez avec nous Marine Le Pen dans moins de 3 minutes, à tout de suite.
Musique
L'actualité c'est aussi aujourd'hui un dernier adieu, dernier adieu à Laurent Fignon, c'était au cimetière du Père Lachaise à Paris.
L'ancien double vainqueur du Tour de France, vaincu par le cancer à 50 ans, a été incinéré en petit comité aujourd'hui.
